tac/project-updates/2025/2025-Q2-Besu.md

```diff
@@ -0,0 +1,142 @@
+parent: 2025
+grand_parent: Project Updates
+---
+
+## Project Health
+
+Hyperledger Besu continues to demonstrate strong project health in Q2 2025, with several significant achievements and milestones reached during this quarter. The project successfully delivered critical infrastructure updates to support the Ethereum Pectra (Prague) hard fork, marking steady commitment to the Ethereum roadmap.
+
+### Major Accomplishments This Quarter
+
+**Pectra Hard Fork Support**: The most significant achievement of Q2 2025 was the successful implementation and deployment of Pectra (Prague) hard fork support. Besu releases 25.4.0, 25.4.1, and 25.5.0 were specifically Pectra Mainnet-ready releases, with the hard fork activating on May 7, 2025. This represents a critical milestone for Ethereum infrastructure and demonstrates Besu's continued role as a key execution client.
+
+**Enhanced Plugin Architecture**: Q2 saw significant improvements to Besu's plugin system, including new plugin providers for block import tracing and expanded transaction permissioning capabilities. These enhancements further solidify Besu's position as a modular and extensible Ethereum client, suitable for use by L2 rollups.
+
+**Performance and Stability Improvements**: The team delivered substantial improvements in block processing performance, peer connectivity optimizations, and RPC response times. Notable enhancements include improved conflict detection in transaction pools and optimized memory management through tuned G1GC configurations.
+
+### Quarterly Release Summary
+
+During Q2 2025, the Besu project released three major versions:
+
+- **Besu 25.4.0** (April): Core Pectra implementation with BLS12-381 precompile support
+- **Besu 25.4.1** (May): Pectra Mainnet-ready release with Prague fork activation
+- **Besu 25.5.0** (June): Post-fork stability and performance optimizations
+
+These releases included over 50 merged pull requests addressing feature development, bug fixes, and security improvements.
+
+## Questions/Issues for the TAC
+
+**Maintainer Diversity Initiative**: The project continues to face challenges with maintainer diversity, with 19 of 21 maintainers working for Consensys and only 2 from Kaleido. The team seeks TAC guidance on establishing formal pathways for increasing organizational diversity among maintainers while maintaining code quality standards.
+
+**Contributor Onboarding**: Recent discussions have highlighted that Besu's contributing guidelines may appear discouraging to new contributors. The team is actively revising documentation to be more welcoming while maintaining quality standards, but seeks best practices from other LF Decentralized Trust projects.
+
+**Enterprise Adoption Tracking**: With the launch of the Besu Certified Service Provider (CSP) program in Q2, the project would benefit from TAC guidance on measuring and reporting enterprise adoption metrics more effectively.
+
+## Releases
+
+### Major Releases (Q2 2025)
```

Comment:
It might be worth referencing the fact that the codebase is being kept healthy by the removal of deprecated features.